packages/manager/src/features/Betas/BetaDetailsList.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import Stack from '@mui/material/Stack';
+import { Beta } from '@linode/api-v4/lib/betas';
+import { AccountBeta } from '@linode/api-v4/lib/account';
+
+import { Paper } from 'src/components/Paper';
+import { Typography } from 'src/components/Typography';
+import { Divider } from 'src/components/Divider';
+
+import BetaDetails from './BetaDetails';
+import { CircleProgress } from 'src/components/CircleProgress';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
+import { APIError } from '@linode/api-v4';
+
+interface Props {
+  title: string;
+  betas: (Beta | AccountBeta)[];
+  isLoading: boolean;
+  errors: APIError[] | null;
+}
+
+const BetaError = (props: { errors: APIError[] | null }) =>
+  props.errors ? <ErrorState errorText={props.errors[0].reason} /> : null;
+
+const BetaLoading = (props: { isLoading: boolean }) =>
+  props.isLoading ? <CircleProgress /> : null;
+
+export const BetaDetailsList = (props: Props) => {
+  const { title, betas, isLoading, errors } = props;
+  return (
+    <Paper>
+      <Typography variant="h2">{title}</Typography>
+      <Divider spacingBottom={20} spacingTop={20} />
+      <BetaError errors={errors} />
+      <BetaLoading isLoading={isLoading} />
```

Comment:
Unnecessary abstraction. 

```suggestion
      {isLoading && <CircleProgress />}
```